src/main/java/mate/controller/car/GetAllCarsController.java

```diff
@@ -1,25 +1,17 @@
 package mate.controller.car;
 
 import java.io.IOException;
-import java.util.List;
 import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import mate.lib.Injector;
-import mate.model.Car;
-import mate.service.CarService;
 
+@WebServlet(urlPatterns = "/cars/")
 public class GetAllCarsController extends HttpServlet {
```

Comment:
We don't need to change this controller. We should still have a possibility to get a list of all the cars.